project-maintainers.csv

```diff
@@ -624,21 +624,28 @@ Incubating,Contour,Steve Kriss,VMware,skriss,https://github.com/projectcontour/c
 ,,Tero Saarni,Ericsson,tsaarni,
 ,,Sunjay Bhatia,VMware,sunjayBhatia,
 ,,Nigel Brown,VMware,pnbrown,
-Incubating,Operator Framework: SDK,Eric Stroczynski,Red Hat,estroz,https://github.com/operator-framework/operator-sdk/blob/master/.cncf-maintainers
-,,Joe Lanford,Red Hat,joelanford,
+Incubating,Operator Framework: SDK,Joe Lanford,Red Hat,joelanford,,https://github.com/operator-framework/operator-sdk/blob/master/.cncf-maintainers
 ,,Ish Shah,Red Hat,theishshah,
-,,Camila Macedo,Red Hat,camilamacedo86,
-,,Jesus Rodriguez,Red Hat,jmrodri,
-,,Austin Macdonald ,Red Hat,asmacdo,
 ,,Jonathan Berkhahn,IBM,jberkhahn,
 ,,Fabian von Feilitzsch,Red Hat,fabianvf,
 ,,Varsha Narsing,Red Hat,varshaprasad96,
 ,,Rashmi Gottipati,Red Hat,rashmigottipati,
-Incubating,Operator Framework: Lifecycle Manager,Evan Cordell,Red Hat,ecordell,https://github.com/operator-framework/operator-lifecycle-manager/blob/master/OWNERS
-,,Nick Hale,Red Hat,njhale,

```

Comment:
Just wanted to notify @njhale and @ecordell that I'm planning to remove them from the list of maintainers.